app/templates/travisci-vs-jenkins.hbs

```diff
@@ -1,9 +1,11 @@
-<TravisLayout @layoutName="layouts/center" class="tailwind-overrides">
+<TravisLayout @layoutName="layouts/striped" class="tailwind-overrides">
```

Comment:
Needed to switch, for full-width sections with a border between the compare and trust sections